DEVELOP.rst

```diff
@@ -117,9 +117,10 @@ After that, tests can be run with:
 
 ::
 
-    $ cd tests
```

Comment:
Due to [testpaths](https://docs.pytest.org/en/6.2.x/reference.html#confval-testpaths) being added